hooks/useOnboard.tsx

```diff
@@ -4,41 +4,32 @@ import Onboard from "bnc-onboard";
 
 import { useConnection } from "./useConnection";
 import { onboardBaseConfig } from "../utils/constants";
-import { UnsupportedChainIdError, isValidChainId } from "../utils/chainId";
+import { ChainId } from "../utils/chainId";
 
-export function useOnboard() {
+export function useOnboard(chainId?: ChainId) {
   const { connect, disconnect, update, setError } = useConnection();
   const instance = React.useMemo(
     () =>
       Onboard({
-        ...onboardBaseConfig(),
+        ...onboardBaseConfig(chainId),
```

Comment:
i wonder if the problem here is that onboard keeps being reloaded during the lifetime of the page. we ran into this issue in across, and it created some wierd behavior

Reply:
Hmm, interesting. I didn't wasn't aware of these weird behaviour. Should I undo this change? I think Onboard will work just fine if I don't pass the chain id of the user's wallet